src/System.Net.Primitives/tests/FunctionalTests/CookieTest.cs

```diff
@@ -239,6 +239,13 @@ public static void Value_GetSet_Success()
             c.Value = null;
             Assert.Equal(string.Empty, c.Value);
         }
+        
+        [Fact]
```

Comment:
Please add a comment specifying that this is not expected (and will fail) in .Net Desktop.